src/configuration/locales/en/translations.json

```diff
@@ -1,3 +1,33 @@
 {
-  "Take the survey!": "Take the survey!"
+  "...Loading": "...Loading",
+  "Beekeeper": "Beekeeper",
+  "Beginner": "Beginner",
+  "Billed: {{value}}": "Billed: {{value}}",
+  "Description {{description}}:": "Description: {{description}}",
+  "Experience: {{experience}}": "Experience: {{experience}}",
+  "Expert": "Expert",
+  "FreeSpace": "Free space available",
+  "Garden": "Garden",
+  "HAPPYCULTEUR © COPYRIGHT 2019. ALL RIGHTS RESERVED.": "Happyculteur © COPYRIGHT 2019. ALL RIGHTS RESERVED.",
+  "Happyculteur partner!": "Happyculteur partner!",
+  "Individual": "Individual",
+  "Install": "Install",
+  "Interest:": "Interest:",
```

Comment:
What do you think of `Interest` in plural: `Interests ?``
EDIT: or create another local for the plural. As it seems users have many interests